app/containers/App.js

```diff
@@ -117,15 +125,23 @@ export class App extends Component {
   setServer = info => {
     console.log('will use', info.label, 'server');
     var serverinfo = serverdata[info.label];
-    this.props.route.api.setHosts(serverinfo);
+    this.props.api.setHosts(serverinfo);
     this.setState({server: info.label});
   };
 
   render() {
     return (
       <div className={styles.app} onClick={this.handleDismissDropdown}>
         <Header location={this.props.location} />
```

Comment:
Please excuse the spacing below, I got my editor sorted out and untabified everything later.